typescript-carousel/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>TypeScript Carousel</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="arrows left">
+          <i class="fa-solid fa-chevron-left" id="left"></i>
+        </div>
+        <div class="hidden" id="img-2">
+          <img src="images/001.png" alt="Bulbasuar" />
+        </div>
+        <div class="hidden" id="img-3">
+          <img src="images/004.png" alt="Charmander" />
+        </div>
+        <div class="hidden" id="img-4">
+          <img src="images/007.png" alt="Squirtle" />
+        </div>
+        <div class="unhidden" id="img-1">
+          <img src="images/025.png" alt="Pikachu" />
+        </div>
+        <div class="hidden" id="img-5">
+          <img src="images/039.png" alt="jigglypuff" />
+        </div>
+        <div class="arrows right">
+          <i class="fa-solid fa-chevron-right" id="right"></i>
+        </div>
+      </div>
+      <div class="row">
+        <div class="img-dots">
+          <span class="filled" id="dot1"></span>
+          <span class="empty" id="dot2"></span>
+          <span class="empty" id="dot3"></span>
+          <span class="empty" id="dot4"></span>
+          <span class="empty" id="dot5"></span>
+        </div>
+      </div>
```

Comment:
This `row` doesn't do anything here. It would _flex_ it's children in a row (horizontal line), but it only works on the _direct_ children, not the descendants so that `span` elements are not affected. You probably did not notice since `span` elements are `inline` and can share the same line.